operator/roles/kiali-deploy/tasks/main.yml

```diff
@@ -209,28 +216,28 @@
 
 - name: Determine all accessible namespaces, expanding regex expressions to matched namespaces
   set_fact:
-    all_accessible_namespaces: "{{ ((all_accessible_namespaces | default([ kiali_vars.deployment.namespace ])) + [ item.1 | regex_search(item.0) ]) | unique }}"
```

Comment:
I'm sorry - I'm trying to be as clear as I can possibly be... I do not know how else to request this other than repeat what I have been saying the last few times.

Please also revert back all your unnecessary changes to these double quotes found in this yaml file. They have been changed to single quotes and create unnecessary changes that should not be here. Please review your [diff](https://github.com/kiali/kiali/pull/1253/files) - notice all the double quotes that have been changed to single quotes all throughout this file. Revert those changes - keep the double quotes that were originally there.

As per my previous comments:

From 2 weeks ago: https://github.com/kiali/kiali/pull/1253#pullrequestreview-264261712

`Revert all of these changes converting double quotes to single quotes`

From yesterday: https://github.com/kiali/kiali/pull/1253#discussion_r309269602

`Please revert back all ... quotes as they originally were...I see alot of double-quotes getting replaced with single-quotes for some reason.`

Reply:
I copy pasted the file in the master and compared now the only difference is my changes. Hopefully this does it :)